pkgs/development/coq-modules/coq-elpi/default.nix

```
@@ -6,20 +6,21 @@ with builtins; with lib; let
     { case = "8.12"; out = { version = "1.12.0"; };}
     { case = "8.13"; out = { version = "1.13.7"; };}
     { case = "8.14"; out = { version = "1.13.7"; };}
-    { case = "8.15"; out = { version = "1.14.1"; };}
+    { case = "8.15"; out = { version = "1.15.0"; };}
   ] { version = "1.14.1"; } );
```

```suggestion
  ] {} );
```
so that this can be used with coq-master branches in CIs (and doesn't need to be updated each time elpi itself is updated)